lib/api.js

```diff
@@ -437,7 +437,7 @@ function getTraceSettings (xtrace, options = {}) {
       source: 5,
       rate: 0,
       edge: false,
-      metadata: aob.Metadata.makeRandom(0)
+      metadata: aob.Event.makeRandom(0)
```

Comment:
i think it makes sense to change it. i'm not sure what i want to change it to, so i've punted for now. it's not a "real" event in that it's only the metadata portion of an event. i'm thinking something like "precursor" or "taskIdSource" or something similar but a little more abstract. the whole purpose of it is to supply the task ID. it might have come from an inbound x-trace id or it might just be random data when starting a new trace.

any ideas? it's part of the documented api so, while i'm breaking other things and now would be a good time to change this, i don't want to change this unless i think there is a significantly better name. and metadata is kind of abstract as long as we don't think of it in terms of the previous definition.

maybe "metataskId"?

Reply:
I like "traceTaskId"

Reply:
me too.